google-cloud-storage/src/test/java/com/google/cloud/storage/conformance/retry/RpcMethodMappings.java

```diff
@@ -1130,11 +1140,21 @@ private static void get(ArrayList<RpcMethodMapping> a) {
                         (ctx, c) ->
                             ctx.peek(
                                 state -> {
-                                  ReadChannel reader =
-                                      ctx.getStorage().reader(ctx.getState().getBlobId());
-                                  WritableByteChannel write =
-                                      Channels.newChannel(NullOutputStream.INSTANCE);
-                                  ByteStreams.copy(reader, write);
+                                  try {
+                                    ReadChannel reader =
+                                        ctx.getStorage().reader(ctx.getState().getBlobId());
+                                    WritableByteChannel write =
+                                        Channels.newChannel(NullOutputStream.INSTANCE);
+                                    ByteStreams.copy(reader, write);
+                                  } catch (IOException e) {
```

Comment:
Why does this end up being necessary?

Reply:
BlobReadChannel wraps any retry helper exception in an IOException
https://github.com/googleapis/java-storage/blob/master/google-cloud-storage/src/main/java/com/google/cloud/storage/BlobReadChannel.java#L140-L142

We can't change the exception hierarchy without the possibility of breaking customers error handling code, so I opted to unwrap here in the test to get to the actual StorageException/Rpc Exception.